docs/components/miranum-connect/java-client/miranum-worker.md

```diff
@@ -111,3 +111,54 @@ used as input in a subsequent activity of our process.
 :::info
 For an actual example take a look on our [miranum-connect guide](../../../guides/getting-started/pizza-order-miranum.md).
 :::
+
+
+## Generate Camunda Element-Templates
+:::note
+This feature is currently only available for Camunda Platform 7.
+:::
+
+Camunda element-templates are pre-configured building blocks that simplify the process of designing and implementing processes
+in the Camunda Platform. They are used to standardize and streamline the modeling process by providing predefined elements with 
+pre-configured properties, which can be easily customized to meet specific business requirements. This leads to a standardized way 
+of modelling processes across different departments and teams within an organization and accelerates the creation of new process models 
+for organizations that frequently develop new processes.
+
+To use the element-template generation feature for Camunda the `miranum-core` dependency or `element-template-api` dependency 
+needs to be added to the project. Additionally, it is required to add a specific dependency for the Workflow-Engine you are using. 
```

Comment:
That's exactly the thought I had. Ideally we can use remove the Camunda 7 only note and leave the rest as it is. 
Since you are unsure about it as well I would just leave it as it is. 🤔 
But good catch! 👍